demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp

```diff
@@ -40,6 +40,18 @@ static const char delay_message[] = "Optional. Delay between frames used for vis
 static const char output_log_message[] = "Optional. The file name to write output log file with results of pedestrian tracking. "
                                           "The format of the log file is compatible with MOTChallenge format.";
 static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
+static const char at_message[] = "Required. Architecture type: centernet, ssd or yolo";
+static const char thresh_output_message[] = "Optional. Probability threshold for detections.";
+static const char input_resizable_message[] = "Optional. Enables resizable input with support of ROI crop & auto resize.";
+static const char iou_thresh_output_message[] = "Optional. Filtering intersection over union threshold for overlapping boxes.";
+static const char yolo_af_message[] = "Optional. Use advanced postprocessing/filtering algorithm for YOLO.";
+static const char labels_message[] = "Optional. Path to a file with labels mapping.";
+static const char nireq_message[] = "Optional. Number of infer requests. If this option is omitted, number of infer requests is determined automatically.";
+static const char num_threads_message[] = "Optional. Number of threads.";
+static const char num_streams_message[] = "Optional. Number of streams to use for inference on the CPU or/and GPU in "
+"throughput mode (for HETERO and MULTI device cases use format "
+"<device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>)";
+static const char person_label_message[] = "Optional. Label of class person for detector. Default -1 for tracking all objects";
```

Comment:
The description of new parameters should be also added to README file (running section).

Reply:
done